src/index.js

```diff
@@ -143,6 +144,34 @@ export default function ({ template, types }) {
             }
           },
         });
+
+        programPath.scope.crawl();
```

Comment:
ok so what i have thought of is that in my traversal I look for these four types: ObjectProperty, ClassProperty, AssignmentExpression (all 3 same as current) and ImportDeclaration
in the ImportDeclaration function, I look at the bindings of specifier.
I look at its parent and see if its any one of ObjectProperty, ClassProperty or AssignmentExpression and run them against the same function as the above traversal to see if they are propTypes declaration.
If all bindings are propTypes then I remove the importDeclaration node.

Now this means that I will be checking isReactClass twice because I cannot cut it down to just once as some repos might be using both `React.PropTypes` and `import 'prop-types'`